js-extensions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/js-extensions",
-  "version": "0.0.14",
+  "version": "0.0.15-beta4",
```

Comment:
I've taken a look at this PR on my windows machine and I can successfully build the dashboard now but there are several test failures. 

Video(4:30 min) with my repo steps: [https://www.youtube.com/watch?v=KFe-ubQ-VdA?version=3&vq=hd1080]

Build Summary: https://gist.github.com/ianceicys/eafeba18ae28435cdaaf76dc4c7bb385

nohup test failure - https://gist.github.com/anonymous/4759fee0a0218fe9cb5aa4324a2111d1

sh test failure - https://gist.github.com/anonymous/c554be4948c2bd949de1222dc437c005